src/components/ReportActionItem/ReportActionItemImages.js

```diff
@@ -54,10 +54,12 @@ function ReportActionItemImages({images, size, total, isHovered}) {
         <View style={[styles.reportActionItemImages, hoverStyle]}>
             {_.map(shownImages, ({thumbnail, image}, index) => {
                 const isLastImage = index === numberOfShownImages - 1;
+                const shouldShowBorder = shownImages.length > 1 && index < shownImages.length - 1;
```

Comment:
NAB: This logic could have a comment explaining _why_ this logic is here (and why it isn't anything else).